docs/io/unified.rst

```diff
@@ -322,6 +331,140 @@ FITS time standard will be set.  When reading this back into Astropy, the
+and the time coordinate column ``time`` as ``[1, 2]`` will give::
+
+    >>> from astropy.time import Time, TimeDelta
+    >>> from astropy.table import Table
+    >>> tm = Table.read('chandra_events.fits', astropy_native=True)
+    WARNING: Time column "time" reference position will be ignored due to unspecified observatory position. [astropy.io.fits.fitstime]
+    >>> tm['time']
+    <Time object: scale='tt' format='mjd' value=[ 50814.00001157  50814.00002315]>
+    >>> # MJDREF  =  5.0814000000000E+04, TIMESYS = 'TT'
+    >>> ref_time = Time(5.0814000000000E+04, format='mjd', scale='tt')
+    >>> # TTYPE1  = 'time', TUNIT1 = 's'
+    >>> delta_time = TimeDelta([1, 2], format='sec')
+    >>> ref_time + delta_time == tm['time']
+    array([ True,  True], dtype=bool)
+
+By default, FITS table columns will be read as standard `~astropy.table.Column`
+objects without taking the FITS time standard into consideration.
+
+* String time column in ISO-8601 Datetime format
+
+FITS uses a subset of ISO-8601 (which in itself does not imply a particular time scale)
+for several time-related keywords, such as DATE-xxx. Following the FITS standard its
+values must be written as a character string in the following ``datetime`` format:
+
+``[+/-C]CCYY-MM-DD[Thh:mm:ss[.s...]]``
+
+A time coordinate column can be constructed using this representation of time.
+The following is an example of an ISO-8601 ``datetime`` format time column:
+
+.. parsed-literal::
+
+    TIME
+    ----
+    1999-01-01T00:00:00
+    1999-01-01T00:00:40
+    1999-01-01T00:01:06
+    .
+    .
+    .
+    1999-01-20T01:10:00
+
+The criteria for identifying a time coordinate column in ISO-8601 format is as follows:
+
+A time column is identified using the time coordinate frame keywords as described in
+:ref:`fits_time_column`. Once it has been identified, its datatype is checked in order
+to determine its representation format. Since ISO-8601 ``datetime`` format is the only
+string representation of time, a time coordinate column having string datatype will be
+automatically read as a `~astropy.time.Time` object with ``format='fits'`` ('fits'
+represents the FITS ISO-8601 format).
+
+As this format does not imply a particular time scale, it is determined using the time
+scale keywords in the header (``TCTYP`` or ``TIMESYS``) or their defaults. The other time
+coordinate information is also determined in the same way, using the time coordinate
+frame keywords. All ISO-8601 times are relative to a globally accepted zero point
+(year 0 corresponds to 1 BCE) and are thus are not relative to the reference time
+keywords (MJDREF, JDREF or DATEREF). Hence, these keywords will be ignored while dealing
+with ISO-8601 time columns.
+
+.. note::
+
+   Reading FITS files with time coordinate columns *may* fail. Astropy supports
+   a large subset of these files, but there are still some FITS files which are
+   not compliant with any aspect of the standard.
```

Comment:
I think this is a good place to remind users that they should raise an issue if they have such a file; for instance, add "If you have such a file, please don't hesitate to let us know, e.g., by opening an issue in the `issue tracker <https://github.com/astropy/astropy/issues>`_."